templates/storageclass-kadalu.custom.yaml.j2

```diff
@@ -0,0 +1,11 @@
+# -*- mode: yaml -*-
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: kadalu.{{ hostvol_name }}
+provisioner: kadalu
+allowVolumeExpansion: true
+parameters:
+  hostvol_type: {{ type }}
```

Comment:
This can be removed

Reply:
Removed hostvol_type. When the type is removed, how to specify repllica1 or any other required type?